v1.00.01 Hotfix

Spoffy opened the pull request.

Commit:
duty officers facing right way

Commit:
Bra Duty Officers orientation #2kd1epx

duty officers facing right way

Commit:
Fixes altis trashcan #2kd1enb

#168

Commit:
Stops sites from spawning in the blocked areas #2m7rcu9 (#167)

cerebral-314 committed:
Fixes runtime error in wreck recovery

Bad type comparison between location and array

veteran29 commented on mission/functions/systems/sites/fn_sites_generate.sqf:
```diff
@@ -24,13 +24,26 @@ if (_zonesToGenerateIn isEqualType 0) then {
+		};
+	} forEach vn_mf_markers_blocked_areas;
+
+	_result
```
```suggestion
	params ["_position"];
	vn_mf_markers_blocked_areas findIf {_position inArea _x} != -1
```

Spoffy committed:
Fixes logistic_menu.paa popup and updates tutorial text

Spoffy committed:
Fixes 1.2 magazines not showing in arsenal

Spoffy committed:
Adds 1.1 and 1.2 mags to supply officer crates

Spoffy committed:
Adds vehAssetId property to packaged wrecks

Spoffy committed:
Adds changelog for 1.00.01 (all content up to this commit)

Alablm committed:
Merge pull request #145 from Savage-Game-Design/development

Use raw strings to prevent unicode errors

Alablm committed:
Fix syntax error in log creation function

[Array] select {condition} not {condition} select [Array].

Alablm reviewed (changes requested):
https://github.com/Savage-Game-Design/SGD_mikeforce/commit/55169482683b059cea0b37415ce58c0be2858789#commitcomment-78792644

There is a ton of garbage in this commit, for example the camera position changes and the CBA settings data that just doesn't need to be there, as well as all of the type position changes with the SCALAR and STRING data swapping places. This is going to cause merge headaches with the further changes that need to made to this file.

Going to request that this commit be reverted and the four lines that actually need to be edited are manually changed and added to https://github.com/Savage-Game-Design/Mike-Force/pull/146. If you like I can take care of that myself, just let me know.

DanThomas64 commented:
The SCALAR/STRING thing is a result of profiling/normal branch base Arma. With that a side, the camera change is completely unimportant. 

All that to say, it's not going to cause headaches. The changes might look messy but make no difference.

Alablm committed:
The Bra Mission.sqm changes (#146)

Fog too thick on The Bra: Starting and Forecast prams modified, Lines 283-286

Howitzers not configured for respawn: Links to logic made, lines 11346 to 11375.

Increases RTO delay on the bra by moving the module: Module moved, lines 1539-1540

Adds Trash Can for Green Hornets Arsenal

Alablm committed:
Unlocks Altis mission for non DLC owners. (#147)

Spoffy committed:
Tidies blocked area check

Co-authored-by: Filip Maciejewski <veteran29@users.noreply.github.com>

Spoffy committed:
Bumps version number to v1.00.01
